### DISCUSSION TITLE
how to change the color of the compass or square rulers?

### DISCUSSION THREAD

**Discussion by vladQll** [xournalpp/xournalpp#5036] [Q&A]
I would like to know how I can change the colors because having dark backgrounds I can't appreciate the contrast of the colors.
![image](https://github.com/xournalpp/xournalpp/assets/28522011/81424d1d-95f1-4236-9bc5-9a4899ec5545)

**Comment by rolandlo** [comment]
It's currently hardcoded in `SetsquareView::drawGeometryTool` (file `SetsquareView.cpp`). So you can't change the colors without recompiling Xournal++. If you feel like the colors should be configurable open an enhancement request.
